demisto_sdk/commands/validate/file_validator.py

```diff
@@ -635,13 +630,22 @@ def validate_against_previous_version(self, no_error=False):
         Args:
             no_error (bool): If set to true will restore self._is_valid after run (will not return new errors)
         """
-        if self.prev_ver and self.prev_ver != 'master':
-            print_color('Starting validation against {}'.format(self.prev_ver), LOG_COLORS.GREEN)
-            modified_files, _, _, _ = self.get_modified_and_added_files(self.prev_ver)
-            prev_self_valid = self._is_valid
-            self.validate_modified_files(modified_files)
-            if no_error:
-                self._is_valid = prev_self_valid
+        compared_branch_name = self.prev_ver
+        if not self.prev_ver:
+            content_release_branch_id = self.get_content_release_identifier()
+            if not content_release_branch_id:
+                print_warning('could\'t get content\'s release branch ID. Skipping validation.')
+                return
+            else:
+                self.prev_ver = content_release_branch_id
+                compared_branch_name = 'release branch'
```

Comment:
a hardcoded string is not informative, use `content_release_branch_id`

Reply:
done.